addons/mail/static/tests/thread/thread.test.js

```diff
@@ -514,30 +518,30 @@ test("mention 2 different partners that have the same name", async () => {
     );
 });
 
-test("mention a channel on a second line when the first line contains #", async () => {
+test.skip("mention a channel on a second line when the first line contains #", async () => {
```

Comment:
Some tests below are also skipped, not working (yet)? (won't comment again for the other occurrences of added `skip`).